services.rst

```diff
@@ -125,33 +125,40 @@ Note that a successful configuration of the AD service does not enroll the
 system into the AD. To do this, the AD service must be turned ON. To leave the
 AD, simply turn the AD service OFF.
 
-LDAP
-----
+Lightweight Directory Access Protocol (LDAP)
+--------------------------------------------
 
-LDAP is a directory service to connect to LDAP server.
+LDAP is a directory service to connect to a LDAP server. It must be turned ON
+in order to fetch users from the LDAP directory.
 
-In the web-ui, click on *System* tab to go to the *System* view. This also
-serves as the *Services* view, which is selected by default in the left
-sidebar. To configure LDAP, click on the **wrench** icon and submit the form
-with appropriate values as shown below.
+First, the LDAP service needs to be configured: click on the **wrench** icon
+and fill the form with the values corresponding to your LDAP server.
```

Comment:
```suggestion
and fill in the form with the values corresponding to your LDAP server.
```
Minor readability thing that I wasn't going to bother with but I've seen a couple of 'nice-to-have' changes later so noting now as we then get these all done in one go.
Could be 'fill out' if you prefer as apparently more common in American English but I'm more familiar with 'fill in a/the form'.
Reference:
https://english.stackexchange.com/questions/1514/fill-out-a-form-or-fill-in-a-form
We should probably pick one and stick to it :).

Reply:
Agreed on the "fill in". Thanks a lot for catching all these things!